src/components/Navbar.tsx

```diff
@@ -1,4 +1,15 @@
+import cn from 'classnames';
+import { NavLink, useSearchParams } from 'react-router-dom';
+
 export const Navbar = () => {
+  const [searchParams] = useSearchParams();
+
+  const getLinkClass = ({ isActive }: { isActive: boolean }) => (
+    cn('navbar-item', {
+      'has-background-grey-lighter': isActive,
+    })
+  );
```

Comment:
It is better to define this function outside the component so that it is not created every time a new render is performed.